package.json

```diff
@@ -65,10 +63,42 @@
     "typescript": "^2.0.6"
   },
   "scripts": {
-    "postinstall": "bower install",
```

Comment:
Skal vi stadig have package.json selvom vi har en yarn.json? Kan vi ikke få 1 fil?

Reply:
Yarn benytter stadig package.json til at fortælle hvilke packages der skal bruges. Den har så en yarn.lock i stedet for package.lock. "Yarn install [package]" tilføjer også i package.json